source/agora/common/Transaction.d

```diff
@@ -204,7 +204,7 @@ public Transaction newCoinbaseTX (PublicKey address, Amount value = 0)
 
 *******************************************************************************/
 
-public Amount getSumInput (Transaction tx, Output* delegate (Hash hash, size_t index) findOutput)
+public Amount getSumInput (Transaction tx, Output* delegate (Hash hash, size_t index) findOutput) @trusted
```

Comment:
Make it `@safe` by making `findOutput` `@safe`

Reply:
Modified